db/migrate/20241124050607_create_tickets_edit_person_fields.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTicketsEditPersonFields < ActiveRecord::Migration[7.2]
+  def change
+    create_table :tickets_edit_person_fields do |t|
+      t.bigint :tickets_edit_person_id, null: false
```

Comment:
I think this can be `t.references :tickets_edit_person`

Reply:
Done